test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenJsonOptionsTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Swashbuckle.AspNetCore.SwaggerGen.DependencyInjection;
+using Swashbuckle.AspNetCore.SwaggerGen.Test.Fixtures;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test;
+
+public class SwaggerGenJsonOptionsTests
+{
+    [Fact]
+    public static void Ensure_SwaggerGenJsonOptions_Uses_MinimalApi_JsonOptions_When_Overridden()
+    {
+        var services = new ServiceCollection();
+        services.AddSingleton<IWebHostEnvironment, DummyHostEnvironment>();
+        services.AddSwaggerGen();
+        services.AddMvcCore().AddJsonOptions(o => o.JsonSerializerOptions.Converters.Add(new DummyConverter()));
+        services.AddSwaggerGenMinimalApisJsonOptions();
```

Comment:
Could we explicitly configure the options for Minimal APIs to be different and then assert that  `SerializerOptions` is the same instance instead?

The assert right now shows that it isn't the MVC options, but it doesn't prove it's the Minimal APIs one.